main.go

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Jobteaser
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"fmt"
+	"github.com/galdor/go-cmdline"
+	"github.com/jobteaser/circleci-env/circleci"
+	"os"
+)
+
+var token string
+
+func main() {
+	cli := cmdline.New()
+	cli.AddOption("t", "token", "token", "circleci token")
```

Comment:
Could be nice to add, "which you can generate from the .... page."

Reply:
Also, better make the token required rather than providing a default value... because this default value will not work.